lib/statsd.js

```diff
@@ -216,6 +239,11 @@ Client.prototype.send = function (message, tags, callback) {
       message += '|#' + mergedTags.join(',');
     }
   }
+
+  if (this.protocol === 'tcp') {
+    message += '\n';
+  }
```

Comment:
I think this should be in the `sendMessage` method as that one deals with the quirks of the individual transmission protocols. This is an implementation details of sending the message over TCP and IMHO should be encapsulated by `sendMessage`.